eclass/bitcoincore.eclass

```diff
@@ -195,7 +197,7 @@ bitcoincore_common_depend_use() {
 	in_bcc_iuse "$1" || return
 	BITCOINCORE_COMMON_DEPEND="${BITCOINCORE_COMMON_DEPEND} $1? ( $2 )"
 }
-bitcoincore_common_depend_use upnp net-libs/miniupnpc
+bitcoincore_common_depend_use upnp '>net-libs/miniupnpc-1.9.20150915'
```

Comment:
Why `>`? This operator is useful in extremely rare cases, i.e. it indicates that `1.9.20150915-r1` is acceptable. If that's the case, `>=` is more readable. If it's not, you need to use `>=` with the accurate minimum version that is acceptable (or possibly `-r9999...` but that's ugly and unreliable).